request_a_govuk_domain/request/admin/model_admins.py

```diff
@@ -375,7 +375,7 @@ def get_owner(self, obj):
 
     def response_change(self, request, obj):
         if "_approve" in request.POST:
-            if obj.is_approvable():
+            if self.model.is_approvable(obj):
```

Comment:
Why was this changed?

Reply:
obj was Application, using self.model for Review model and passing in Application obj to is_approvable.

Reply:
We may have to investigate this :)
This is the code block that calls this function, so obj is supposed to be review

    def save_model(self, request, obj, form, change):
        super().save_model(request, obj, form, change)
        # Save the application attributes
        if obj.application.status == ApplicationStatus.NEW:
            obj.application.status = ApplicationStatus.IN_PROGRESS
        # Change the owner to be the current user regardless if there is already a user
        # assigned or not
        obj.application.owner = request.user
        obj.application.save()